test/nodetool/test_status.py

```diff
@@ -45,7 +45,7 @@ class StatusQueryTarget(NamedTuple):
                         "effective ownership information is meaningless")
 
 
-def validate_status_output(res, keyspace, nodes, ownership, resolve, effective_ownership_unknown):
+def validate_status_output(res, keyspace, nodes, ownership, resolve, effective_ownership_unknown, token_count_uknown):
```

Comment:
typo.

Reply:
I don't know what is with me and "uknown", I mis-type it all the time.